lib/checkPullRequestReview.js

```diff
@@ -36,9 +36,12 @@ const PTAL_COMMENTS = ['PTAL', 'Please take a look'];
  * @param {import('probot').Octokit.PullsGetReviewResponse} review
  */
 const handleChangesRequested = async (context, pullRequest, review) => {
+  let allLabels = pullRequest.labels.map((label) => {
+    return label.name;
+  });
+
   const reviewer = review.user.login;
   const author = pullRequest.user.login;
-
```

Comment:
Unneeded change.

Reply:
Will remove it in the latest commit Sir.Thanks.